sale_validity_auto_cancel/models/res_company.py

```diff
@@ -13,6 +14,13 @@ class ResCompany(models.Model):
         help="Quotations will be cancelled after the specified number of"
         " days since the expiration date.",
     )
+    default_sale_order_validity_warning_days = fields.Integer(
```

Comment:
```suggestion
    sale_order_validity_warning_days = fields.Integer(
```

As it will be a generic value it is not really a default because you will not be able to change it anywhere else